### PR TITLE
Let struct visibility to be same as module

### PR DESCRIPTION
**Pull request by h-michael**
feature #77

### CODE REVIEW

File: graphql_client_cli/src/generate.rs [hunk 1]
```diff
@@ -21,10 +23,18 @@ pub fn generate_code(
         _ => None,
     };
 
+    let module_visibility = module_visibility.as_ref().map(|s| s.as_str());
+    let module_visibility = match module_visibility {
+        Some("pub") => syn::VisPublic { pub_token: <Token![pub]>::default() }.into(),
+        Some("private") => syn::Visibility::Inherited {},
+        _ => syn::VisPublic { pub_token: <Token![pub]>::default() }.into(),
```
Comment (h-michael) [inline graphql_client_cli/src/generate.rs]: `Restricted(VisRestricted)` needs complex variables.
So I implement pub and private at first for cli.

**Comment by tomhoule** [comment]
We just need a rebase and this will be merged.

**Comment by h-michael** [comment]
@tomhoule I've rebased it!

**Comment by h-michael** [comment]
Nightly build is failed, because of https://github.com/hyperium/hyper/issues/1688.

**Comment by h-michael** [comment]
@tomhoule 
Thank you for taking the time out of your busy schedule to review.

**Pull request merged**